lib/redirect_map.rb

```diff
@@ -0,0 +1,23 @@
+require 'yaml'
+require 'singleton'
+
+class RedirectMap
+  include Singleton
+
+  attr_reader :redirects
```

Comment:
just a way to inspect what has been loaded, which is used by a test, but that's all. Just for testing and debugging.